app/workers/downstream_discard_draft_worker.rb

```diff
@@ -13,10 +13,10 @@ class DownstreamDiscardDraftWorker
   def perform(args = {})
     assign_attributes(args.symbolize_keys)
 
-    current_path = web_content_item.try(:base_path)
+    current_path = edition.try(:base_path)
```

Comment:
Just tried replacing it, got instead `undefined method `base_path' for nil:NilClass` which I wasn't expecting, but would appear that's why it's a `try`.